clouddrift/raggedarray.py

```diff
@@ -380,9 +380,14 @@ def validate_attributes(self):
             if key not in self.attrs_variables:
                 self.attrs_variables[key] = {}
 
-    def to_xarray(self):
+    def to_xarray(self, cast_to_float32: bool = True):
         """Convert ragged array object to a xarray Dataset.
 
+        Parameters
+        ----------
+        cast_to_float32 : bool, optional
+            Cast all variables to float32 (default is True)
```

Comment:
Perhaps change to: "Cast all float64 variables to float32 (Default is True). This option aims at minimizing the size of the output."

Reply:
Does this change the size of the result in the active memory or only when written to disk or both?

Reply:
Can you use the suggestion feature so we can commit the change here directly?

Yes, unless there is bug with xarray's memory management, both size in memory and on disk will be reduced.